symbolication/datadog_uploader.go

```diff
@@ -147,14 +148,17 @@ func (d *DatadogUploader) HandleExecutable(elfRef *pfelf.Reference, fileID libpf
 }
 
 type executableMetadata struct {
-	Arch       string `json:"arch"`
-	GNUBuildID string `json:"gnu_build_id"`
-	GoBuildID  string `json:"go_build_id"`
-	FileHash   string `json:"file_hash"`
-	Platform   string `json:"platform"`
```

Comment:
Is the removal of the `Platform` field expected?

Reply:
Yes, `platform` is not used on backend side (I also removed it from [datdog-ci](https://github.com/DataDog/datadog-ci/pull/1388)).